config/backend-config/backend-config.go

```diff
@@ -176,6 +177,19 @@ type LibraryT struct {
 
 type LibrariesT []LibraryT
 
+type DgSourceTrackingPlanConfigT struct {
+	SourceId            string                 `json:"sourceId"`
+	SourceConfigVersion int                    `json:"version"`
+	Config              map[string]interface{} `json:"config"`
```

Comment:
add another mergedConfig to this struct and add a receiver method to compute it every time everytime when it si empty from config.
This way maps overriding time later can be saved.

Reply:
Done.